### ISSUE TITLE
OpenApiComparer is not implemented

### ISSUE THREAD

**Issue by adrientoub** [microsoft/OpenAPI.NET#414]
Hello,

My team is using this repository to generate our OpenAPI document, and we started using the [OpenApiComparer](https://github.com/microsoft/OpenAPI.NET/blob/master/src/Microsoft.OpenApi/Services/OpenApiComparer.cs) recently in our project. This method returns a value so we believed it was working until recently when after making a change this still returned an empty list.

After looking at the implementation it's quite clear that this method does nothing but it is documented as being a method that "Compares two OpenApiDocuments and returns a list of differences.".

I believe this method should be removed or should throw instead.

After looking a bit more, there seems to be a valid implementation of this in the vnext branch that was never merged into master.

**Comment by darrelmiller** [comment]
Huh!  I had no idea.  I don't whether it is in finished enough to consider merging in.  If it is I can create a PR based on it.

**Comment by Backs** [comment]
Any new information? Looks lite this code is productoin ready and was added 2 year ago, but still not in master

**Comment by darrelmiller** [comment]
This code has been merged into the vnext branch.  I would like to include it in the v1.2 release but there are still some failing tests.  I have marked them as skip for the moment, but they need to be fixed before we can release.

**Comment by FLAMESpl** [comment]
Hi, do you have ETA for complete feature or at least new update? Latest prerelease available on nuget throws StackOverflowException for my swagger doc.

**Comment by darrelmiller** [comment]
My current thinking is to move comparer capability out to a separate package.  There are several contributed solutions and I don't feel like picking a winner.

**Comment by FLAMESpl** [comment]
What is the status of this task? Is it shelved or is it in backlog for considerable future?

**Comment by nikjw-lmk** [comment]
Did you end up moving the comparer capability into another repo? It doesn't appear to be here anymore so was wondering if it can be found somewhere else. Thanks.

**Comment by darrelmiller** [comment]
Hey @nikjw-lmk . I moved the code into a feature branch https://github.com/microsoft/OpenAPI.NET/tree/feature/comparer/src/Microsoft.OpenApi/Services  I haven't moved it out into a separate module.  I have been focused on trying to get the OpenAPIWorkspace into a releasable state.

**Comment by Chrille79** [comment]
Any progress?
